### ISSUE TITLE
Write context skipped if description contains an identifier

### ISSUE THREAD

**Issue by tbabej** [GothenburgBitFactory/taskwarrior#2550]
Originally reported by @initBasti in #2549, it seems that for a context that already has tags defined:

```
work       read  +work or +freelance                                     yes   
           write +work                                                   yes
```

addition of a new task with an unrelated tag is not handled correctly:

```
basti@basti-TUXEDO-Book-XA1510:~/Documents/Programming_Projects/PyProjects/plenty_rait_tool$ task add pro:Work.Administration pri:H test 3 +test
Created task 44.
The project 'Work.Administration' has changed.  Project 'Work.Administration' is 99% complete (2 of 238 tasks remaining).
basti@basti-TUXEDO-Book-XA1510:~/Documents/Programming_Projects/PyProjects/plenty_rait_tool$ task 44 information

Name          Value                                                      
ID            44                                                         
Description   test 3                                                     
Status        Pending                                                    
Project       Work.Administration
Entered       2021-07-31 19:28:47 (4s)                                   
Last modified 2021-07-31 19:28:47 (4s)
Tags          test                                                       
Virtual tags  LATEST PENDING PRIORITY PROJECT READY TAGGED UDA UNBLOCKED
UUID          a30bfa3c-acff-4a47-a9eb-8818548ec323                       
Urgency        7.8
Priority      H                                                          

    project             1 *    1 =      1
    tags              0.8 *    1 =    0.8
    UDA priority.H      1 *    6 =      6
                                   ------
                                      7.8

Filter: ( ( ( id == 44 ) ) )
basti@basti-TUXEDO-Book-XA1510:~/Documents/Programming_Projects/PyProjects/plenty_rait_tool$ task context

Name       Type  Definition                                              Active
freetime   read  +home or +free or +relax                                no    
           write +home or +free or +relax                                no
open-study read  +free or +study or +programming or +coding or +hacking  no    
           write +free or +study or +programming or +coding or +hacking  no
study      read  +study or +programming and -work or +coding or +hacking no    
           write +study or +programming and -work or +coding or +hacking no
work       read  +work or +freelance                                     yes   
           write +work                                                   yes

Use 'task context none' to unset the current context.
```

**Comment by tbabej** [comment]
The underlying cause here is that the number `3` in `test 3` description is interpreted as UUID and a context break is applied (this is present so that `task 3 info` works even if task ID 3 is outside of the context filter).

```
Applying context 'work'
Detected context string: +work
UUID/ID argument found '3', not applying context.
```

However, applying this break for `write` context does not make sense. Good catch @initBasti !
